active/ZEP0000.md

```diff
@@ -177,13 +178,10 @@ or not taking care of Zarr [CODE OF CONDUCT].
 
 As soon as the draft ZEP is committed to the ZEP repository, the author(s)
 should create a discussion thread for the ZEP to provide a central place to
-discuss and review its contents. The ZEP author(s) may select the GitHub
-discussion feature in the organisation [discussions] or the ZEP repository and
-similarly use the discussion feature in the [zarr-specs] repository for
-specification ZEPs. Alternatively, the author(s) may open a new issue in the
-Zarr [community] repository or start the discussion in the Zarr [Gitter] channel.
-The discussion regarding the ZEP should follow Zarr’s [CODE OF CONDUCT] at all
-times.
+discuss and review its contents. The ZEP author(s) may create an issue in
```

Comment:
Reading the new changes, I think I need to have an overview of the total number of PRs/issues that get created for a ZEP (barring discussion issues that arise due to reviewer feedback).

Reply:
Added a concluding statement at the end of the `Submitting a ZEP` section, which states how many PRs are needed to submit ZEP.